README.md

```diff
@@ -1,33 +1,63 @@
 
 # ILS Middleware
+
 Minimal viable product (MVP) for using [Apache Airflow][AF] to manage Blue Core workflows
 that interact with institutional integrated library systems (ILS) and/or
 library services platform (LSP). Currently there are Directed Acyclic Graphs (DAG)
 for Stanford and Cornell Sinopia-to-ILS/LSP workflows. Alma users with the Linked Data API enabled can use the Alma DAG for connecting Sinopia to Alma.
 
-## Running Locally with Docker
+---
+
+## 🐳 Running Locally with Docker
 Based on the documentation, [Running Airflow in Docker](https://airflow.apache.org/docs/apache-airflow/stable/start/docker.html).
 
-> **NOTE** Make sure there is enough RAM available locally for the
+>⚠️ **NOTE**: Make sure there is enough RAM available locally for the
 > docker daemon, we recommend at least 5GB.
 
-1. Clone repository `git clone https://github.com/blue-core-lod/bluecore-workflows`
-1. If it's commented out, uncomment the line `- ./dags:/opt/airflow/dags` in docker-compose.yaml (under `volumes`, under `x-airflow-common`).
-1. Run `docker compose up airflow-init` to initialize the Airflow
-1. Bring up airflow, `docker compose up` to run the containers in the foreground,
-   add `docker compose up -d` to run as a daemon.
-1. Access Airflow locally at http://localhost:8080
-
-### Setup the SQS queue in localstack for local development
-
-```
-AWS_ACCESS_KEY_ID=999999 AWS_SECRET_ACCESS_KEY=1231 aws sqs \
-    --endpoint-url=http://localhost:4566 create-queue \
-    --region us-west-2 \
-    --queue-name all-institutions
+1. Clone repository: `git clone https://github.com/blue-core-lod/bluecore-workflows`
+2. If it's commented out, uncomment the line `- ./dags:/opt/airflow/dags` in docker-compose.yaml (under `volumes`, under `x-airflow-common`).
```

Comment:
@jermnelson should the section about `- ./dags:/opt/airflow/dags` be removed? 
I didn't notice this in the compose file and wondered if it was deprecated.

Reply:
Sorry, I meant there is no section in the code where i see `- ./dags:/opt/airflow/dags` commented out or not.
The Readme make it sound like its present somewhere?